src/systemd/sd-dhcp-static-lease.h

```diff
@@ -0,0 +1,21 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+
+#include <inttypes.h>
+#include <sys/types.h>
+
+#include "_sd-common.h"
+
+extern const struct hash_ops dhcp_static_leases_hash_ops;
+
+_SD_BEGIN_DECLARATIONS;
+
+typedef struct sd_dhcp_static_lease sd_dhcp_static_lease;
+typedef struct DHCPClientId DHCPClientId;
```

Comment:
the sd-*.h headers are supposed to be public API one day, and such public APIs should only expose structures and symbols prefixed with `sd_`. 

This one is not of that type. Moreover it's an opaque structure afaics, so how is anyone supposed to be able to use this?